service/controller/cluster.go

```diff
@@ -64,7 +66,7 @@ func NewCluster(config ClusterConfig) (*Cluster, error) {
 			Watcher: config.G8sClient.ProviderV1alpha1().AzureConfigs(""),
 
 			RateWait:     informer.DefaultRateWait,
-			ResyncPeriod: informer.DefaultResyncPeriod,
+			ResyncPeriod: 3 * time.Minute,
```

Comment:
I am not sure about this. When I test this makes a lot of sense the the cluster is updated faster. 5 minutes are sometimes super long especially on cluster creation and updates.